hirefire/procs/celery.py

```diff
@@ -1,22 +1,31 @@
 from __future__ import absolute_import
 from collections import Counter
 from itertools import chain
+from logging import getLogger
 
 from celery.app import app_or_default
 
 from ..utils import KeyDefaultDict
 from . import Proc
 
 
+logger = getLogger(__name__)
```

Comment:
It's a matter of taste, but what do you think about using only one logger name, for the entire app. Like `hirefire`. That way it is easier to configure.